contracts/staking/StakingV1_1.sol

```diff
@@ -27,6 +29,83 @@ contract StakingV1_1 is StakingState, StakingConstants {
         _;
     }
 
+    function getCurrentFeeTokens()
+        external
+        view
+        returns (address[] memory)
+    {
+        return currentFeeTokens;
+    }
+
+    function setStakingRewards(address asset, uint256 value)
+        external
+    {
+        //!!!! Enable after FEES_EXTRACTOR is deployed
+        //require(msg.sender == FEES_EXTRACTOR, "unauthorized");
```

Comment:
we need a way to upgrade fee extractor, fee_extractor has basically no storage. if community will want to change fee structure we can just replace fee extractor